tests/libfuzzer/bpf2c_fuzzer/bpf2c_fuzzer.vcxproj.filters

```diff
@@ -45,4 +48,15 @@
       <Filter>Source Files</Filter>
     </CustomBuild>
   </ItemGroup>
+  <ItemGroup>
+    <Object Include="..\..\sample\bad\bad1.o">
+      <Filter>Malformed Object Files</Filter>
+    </Object>
+    <Object Include="..\..\sample\bad\bad2.o">
+      <Filter>Malformed Object Files</Filter>
+    </Object>
+    <Object Include="..\..\sample\bad\bad3.o">
```

Comment:
Why is bad4.o not in this list?

Reply:
Files have been renamed to a uniquely identify each malformed file and additional regression tests have been to verify that bpf2c and verifier are able to correctly detect and reject these files.